fastseq_cli/transformers_generate.py

```diff
@@ -148,16 +148,48 @@ def run(self):
                 self.data_queue.put((-1, POSTPROCESS_FINISHED, None))
                 break
             else:
-                dec = self.tokenizer.batch_decode(summaries,
-                        skip_special_tokens = self.skip_special_tokens,
-                        clean_up_tokenization_spaces =
-                        self.clean_up_tokenization_spaces)
+                no_scores = scores is not None and np.all(np.isnan(scores))
+                if (len(summaries.shape) == 3):
+                    bsz, num_ret_seq, seq_len = summaries.shape
+                    dec = []
+                    new_scores = []
+                    for i in range(bsz):
+                        current = summaries[i,:,:].reshape([num_ret_seq, seq_len])
+                        cur_dec = []
+                        for j in range(num_ret_seq):
+                            cur_dec += [self.tokenizer.decode(summaries[i,j,:],
+                                skip_special_tokens = self.skip_special_tokens,
+                                clean_up_tokenization_spaces =
+                                self.clean_up_tokenization_spaces)]
```

Comment:
The code is not easy to read. Could you please change the formatting (e.g., indent) a little bit?